src/client/components/Header.tsx

```diff
@@ -37,6 +41,28 @@ const Header = React.forwardRef(function (
       justifyContent={'center'}
       alignItems={'center'}
     >
+      <Icon sx={{width:'35px',height:'35px'}}>
```

Comment:
Would like to see this use the MUI toggle component, but feel ok with this for now as we should have more changes coming if we get a new figma design